src/unittest/test_random.cpp

```diff
@@ -64,6 +64,16 @@ void TestRandom::testPseudoRandom()
 
 	for (u32 i = 0; i != 256; i++)
 		UASSERTEQ(int, pr.next(), expected_pseudorandom_results[i]);
```

Comment:
Shouldn't these be `s32` rather than `int`?

Reply:
no effective difference the only risk we have is that `sizeof(int) > sizeof(s32)` but I'll change it